rebar.config

```diff
@@ -2,7 +2,7 @@
 
 {erl_opts,
  [debug_info,
-  {platform_define, "^19.|^2", 'ERLANG_OTP_VERSION_19'},
+%  {platform_define, "^19.|^2", 'ERLANG_OTP_VERSION_19'},
```

Comment:
We are currently on 22.

Reply:
Can we just remove the comment, or do you want something like:
`{platform_define, "^22", 'ERLANG_OTP_VERSION_22'},`

Reply:
Also, if we plan to try to merge into their library in the future, do we want to change this?  They may want to support more platforms?

Reply:
Done.

Reply:
I think the line restricts platforms to OTP 19.  I removed it so that it will run on whatever platform.  I tested it on 22 and 24.